source/shared/_cookie_policy_script.html.erb

```diff
@@ -0,0 +1,31 @@
+<body onload="checkCookie()"></body>
+<script>
+  function setCookie(cname, cvalue) {
+    document.cookie = cname + "=" + cvalue;
+  }
+
+  function getCookie(cname) {
+    var name = cname + "=";
+    var decodedCookie = decodeURIComponent(document.cookie);
+    var ca = decodedCookie.split(';');
```

Comment:
Would be easier to read if these variable names were more descriptive. Eg: `cookieName` rather than `cname`. I'm not sure what `ca` stands for, then we have `ca` and `c` below. All could do with more descriptive names.

Reply:
Yeah ill make them more descriptive.